iib/workers/tasks/build.py

```diff
@@ -337,6 +341,101 @@ def _get_resolved_image(pull_spec):
     return pull_spec_resolved
 
 
+@retry(attempts=2, wait_on=IIBError, logger=log)
+def _get_present_bundles(from_index, base_dir):
+    """
+    Get a list of bundles already present in the index image.
+
+    :param str from_index: index image to inspect
+    :param base_dir: base directory to create temporary files in
+    :return: list of tuples containing bundle names and tags
+    :raises IIBError: if any of the commands fail
```

Comment:
Return type can be added here `:rtype: list(tuple)`.

Reply:
Done.